lib/ui/views/installer/installer_viewmodel.dart

```diff
@@ -453,6 +454,7 @@ class InstallerViewModel extends BaseViewModel {
   }
 
   Future<void> installResult(BuildContext context, bool installAsRoot) async {
```

Comment:
I was lazy to comprehend all the `try-catch` thing, so I've set `isInstalling` in each reasonable line in `installResult` method.